OrbitGl/TrackManager.h

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#pragma once
+
+#include <stdint.h>  // for int32_t, uint64_t, uint32_t
+#include <stdlib.h>  // for abs
+
+#include <map>     // for map
+#include <memory>  // for shared_ptr
+#include <mutex>   // for recursive_mutex
+#include <string>  // for string
+#include <unordered_map>
+#include <vector>  // for vector
+
+#include "AsyncTrack.h"
+#include "FrameTrack.h"
+#include "GpuTrack.h"
+#include "GraphTrack.h"
+#include "PickingManager.h"  // for PickingMode
+#include "SchedulerTrack.h"
+#include "StringManager.h"  // for StringManager
+#include "ThreadTrack.h"
+#include "Timer.h"            // for Timer
+#include "Track.h"            // for Track
+#include "capture_data.pb.h"  // for FunctionInfo
+
+class TimeGraph;
```

Comment:
Right. So basically keep this forward declaration and remove the other one. But also add a bug for that. Mid-term, expose all the information that we use from TimeGraph directly (maybe even as parameter instead of having it as a member). Also isn't too much.

Reply:
Again here, sorry for the other forward declaration. And about this one. I wrote my next steps here, which I'm not planning to do all of them, but erasing TimeGraph dependence in TrackManager and Tracks is my P0, because I wanted to make tests after that. Given the amount of comments which I received, I will do another PR and therefore, write a TODO here.

Reply:
And thanks for the link florianK :)

Reply:
b/176086740